get-new-tool-versions/verify-new-tool-version-added-to-image.ps1

```diff
@@ -0,0 +1,43 @@
+<#
+.SYNOPSIS
+Check and return list of new available tool versions
+
+.PARAMETER ToolName
+Required parameter. The name of tool for which parser is available (Node, Go, Python, Xamarin)
+#>
+
+param (
+    [Parameter(Mandatory)] [string] $ToolName
+)
+
+if ($ToolName -eq "Python") {
+    $builtStableMinorVersions = ((Invoke-RestMethod "https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json") |
+        Where-Object {$_.stable -eq $true}).version |
+        ForEach-Object {$_.split(".")[0,1] -join(".")} |
+        Select-Object -Unique
+    $latestExistingMinorVesion = ((Invoke-RestMethod "https://raw.githubusercontent.com/actions/virtual-environments/main/images/win/toolsets/toolset-2019.json").toolcache |
```

Comment:
nitpick: let's split on two parts this long statement

Reply:
Updated